src/pages/my_page/MyPagePage.tsx

```diff
@@ -55,95 +59,133 @@ const MyPagePage = () => {
     <StyledPageWithSideMenu>
       <SidePanel aria-labelledby="my-page-title">
         <SideNavHeader icon={FavoriteBorderIcon} text={t('my_page.my_page')} id="my-page-title" />
-
-        <NavigationList>
+        <Box sx={{ display: 'flex', flexDirection: 'column' }}>
           {user?.isCreator && [
-            <LinkButton
-              key={dataTestId.myPage.messagesLink}
-              data-testid={dataTestId.myPage.messagesLink}
-              isSelected={currentPath === UrlPathTemplate.MyPageMessages}
-              to={UrlPathTemplate.MyPageMessages}
-              startIcon={<ChatBubbleOutlineOutlinedIcon />}>
-              {t('my_page.messages.messages')}
-            </LinkButton>,
-            <Divider key="divider1" />,
-            <LinkButtonRow key={dataTestId.myPage.myRegistrationsLink}>
+            <NavigationListAccordion
+              dataTestId={dataTestId.myPage.messagesAccordion}
+              title={t('my_page.messages.messages')}
+              startIcon={<ChatBubbleIcon fontSize="small" />}
+              accordionPath={UrlPathTemplate.MyPageMessages}
+              defaultPath={UrlPathTemplate.MyPageMyMessages}>
+              <NavigationList>
+                <LinkButton
+                  key={dataTestId.myPage.messagesLink}
+                  data-testid={dataTestId.myPage.messagesLink}
+                  isSelected={currentPath === UrlPathTemplate.MyPageMyMessages}
+                  to={UrlPathTemplate.MyPageMyMessages}>
+                  {t('my_page.messages.messages')}
+                </LinkButton>
+              </NavigationList>
+            </NavigationListAccordion>,
+
+            <NavigationListAccordion
+              title={t('common.registrations')}
```

Comment:
Mangler keys nedover her
![bilde](https://user-images.githubusercontent.com/6313468/236781193-395c687b-cf4f-47ae-85be-cc2bf03ceb6c.png)


```suggestion
              key={dataTestId.myPage.registrationsAccordion}
              title={t('common.registrations')}
```